Makefile

```diff
@@ -33,6 +33,7 @@ clean:
 clean-vms:
 	@echo DO NOT USE THIS COMMAND UNLESS YOU ABSOLUTELY HAVE TO. PRESS CTRL-C NOW.
 	@sleep 20
+	pkill -9 VBoxHeadless
```

Comment:
Won't this nuke all running headless VirtualBox VMs, including those which don't belong to volplugin?